install/on_wsl.md

```diff
@@ -7,4 +7,4 @@ permalink: /install/on_wsl/
 
 The Crystal compiler doesn't run on Windows _yet_. But Crystal can be used with the [Windows Subsystem for Linux](https://msdn.microsoft.com/en-us/commandline/wsl/about), a compatibility-layer for Linux executables running natively on Windows 10.
 
-Several Linux distribution are supported by WSL and the installation instructions for Crystal are equal to that of the respective Linux distribution, most commonly [Debian](on_debian) and [Ubuntu](on_ubuntu).
+Several Linux distribution are supported by WSL and the installation instructions for Crystal are equal to that of the respective Linux distribution, most commonly [Debian](/install/on_debian) and [Ubuntu](/install/on_ubuntu).
```

Comment:
/cc @girng

Reply:
`../on_debian` would work, too. Editing a `makefile` to fix broken links on a website seems a bit extreme, no?  
No wonder why JavaScript/HTML is hated by everyone, everyone over-complicates web development (ambiguous permalink: /install/on_wsl/, editing a makefile, editing markdown). When the only thing that needs to happen is a simple href change inside a `a` tag.

Reply:
What? The Makefile is not required for the fix. It just adds a tool to ensure links don't get broken. It's essentially a QA feature and has already helped identify other broken links besides the ones you reported. 
This doesn't even involve any JavaScript and ai can't make anything of the rest of your rant. This is literally just a simple href change.

Reply:
You don't need a tool.. to make sure "links don't get broken". That's another added layer of complexity. You literally just change the href links if they are broken, it's really that simple.

Reply:
So how do you know they are broken and need to be fixed? Do I manually iterate all 900+ links on the website?